stl/inc/regex

```diff
@@ -3900,20 +3904,21 @@ int _Parser<_FwdIt, _Elem, _RxTraits>::_Do_digits(int _Base, int _Count) { // tr
 }
 
 template <class _FwdIt, class _Elem, class _RxTraits>
-bool _Parser<_FwdIt, _Elem, _RxTraits>::_DecimalDigits() { // check for decimal value
-    return _Do_digits(10, INT_MAX) != INT_MAX;
+bool _Parser<_FwdIt, _Elem, _RxTraits>::_DecimalDigits(
+    regex_constants::error_type _ErrorType) { // check for decimal value
+    return _Do_digits(10, INT_MAX, _ErrorType) != INT_MAX;
 }
 
 template <class _FwdIt, class _Elem, class _RxTraits>
 void _Parser<_FwdIt, _Elem, _RxTraits>::_HexDigits(int _Count) { // check for _Count hex digits
-    if (_Do_digits(16, _Count) != 0) {
+    if (_Do_digits(16, _Count, regex_constants::error_escape) != 0) {

```

Comment:
Noting here that `_HexDigits` will never fail in this way (and neither will `_OctalDigits`) - no changes requested.